docs/node-configuration.md

```diff
@@ -347,6 +348,10 @@ where:
 - `MaxWebSocketFeeds` -- the maximum simultaneous event subscriptions number
   for a single client (16 by default). Attemps to create additional subscriptions
   will lead to error.
+- `MempoolSubscriptionsEnabled` - enables or disables mempool_event WebSocket
+  notifications (for transactions being added to or removed from the mempool).
+  Defaults to false, to avoid running the subscriptions dispatcher by default,
+  as it may slightly affect the node's performance.
```

Comment:
Remove `by default,`.

s/`may slightly affect the node's performance`/`can noticeably affect the node's performance`